org.freedesktop.LinuxAudio.Plugins.Cardinal.yml

```
@@ -0,0 +1,51 @@
+id: org.freedesktop.LinuxAudio.Plugins.Cardinal
+runtime: org.freedesktop.LinuxAudio.BaseExtension
+sdk: org.freedesktop.Sdk//21.08
+runtime-version: '21.08'
+build-extension: true
+appstream-compose: false
+branch: '21.08'
+build-options:
+  prefix: /app/extensions/Plugins/Cardinal
+  prepend-pkg-config-path: /app/extensions/Plugins/Cardinal/lib/pkgconfig
+  prepend-ld-library-path: /app/extensions/Plugins/Cardinal/lib
+modules:
+  - name: jansson
+    buildsystem: cmake-ninja
+    config-opts:
+      - '-DJANSSON_BUILD_DOCS=OFF'
+    sources:
+      - type: archive
+        url: https://github.com/akheron/jansson/releases/download/v2.14/jansson-2.14.tar.bz2
+        sha256: fba956f27c6ae56ce6dfd52fbf9d20254aad42821f74fa52f83957625294afb9
+  - name: Cardinal
+    buildsystem: simple
+    build-commands:
+      - 'make PREFIX=${FLATPAK_DEST} SYSDEPS=true -j $FLATPAK_BUILDER_N_JOBS'
+      - 'make PREFIX=${FLATPAK_DEST} install'
+    post-install:
+      - 'ln -s ${FLATPAK_DEST}/share/cardinal ${FLATPAK_DEST}/lib/lv2/Cardinal.lv2/resources'
+      - 'ln -s ${FLATPAK_DEST}/share/cardinal ${FLATPAK_DEST}/lib/lv2/CardinalFX.lv2/resources'
+      - 'ln -s ${FLATPAK_DEST}/share/cardinal ${FLATPAK_DEST}/lib/lv2/CardinalSynth.lv2/resources'
+      - 'ln -s ${FLATPAK_DEST}/share/cardinal ${FLATPAK_DEST}/lib/vst/Cardinal.vst/resources'
+      - 'ln -s ${FLATPAK_DEST}/share/cardinal ${FLATPAK_DEST}/lib/vst3/Cardinal.vst3/Contents/Resources'
+      - 'ln -s ${FLATPAK_DEST}/share/cardinal ${FLATPAK_DEST}/lib/vst3/CardinalFX.vst3/Contents/Resources'
+      - 'ln -s ${FLATPAK_DEST}/share/cardinal ${FLATPAK_DEST}/lib/vst3/CardinalSynth.vst3/Contents/Resources'
+      - 'install -d ${FLATPAK_DEST}/lv2'
+      - 'install -d ${FLATPAK_DEST}/lxvst'
+      - 'install -d ${FLATPAK_DEST}/vst3'
+      - 'ln -s ${FLATPAK_DEST}/lib/lv2/Cardinal.lv2 ${FLATPAK_DEST}/lv2/Cardinal.lv2'
+      - 'ln -s ${FLATPAK_DEST}/lib/lv2/CardinalFX.lv2 ${FLATPAK_DEST}/lv2/CardinalFX.lv2'
+      - 'ln -s ${FLATPAK_DEST}/lib/lv2/CardinalSynth.lv2 ${FLATPAK_DEST}/lv2/CardinalSynth.lv2'
+      - 'ln -s ${FLATPAK_DEST}/lib/vst/Cardinal.vst ${FLATPAK_DEST}/lxvst/Cardinal.vst'
+      - 'ln -s ${FLATPAK_DEST}/lib/vst3/Cardinal.vst3 ${FLATPAK_DEST}/vst3/Cardinal.vst3'
+      - 'ln -s ${FLATPAK_DEST}/lib/vst3/CardinalFX.vst3 ${FLATPAK_DEST}/vst3/CardinalFX.vst3'
+      - 'ln -s ${FLATPAK_DEST}/lib/vst3/CardinalSynth.vst3 ${FLATPAK_DEST}/vst3/CardinalSynth.vst3'
```

```suggestion
      - ln -s lib/lv2 ${FLATPAK_DEST}/lv2
      - ln -s lib/vst ${FLATPAK_DEST}/vst
      - ln -s vst ${FLATPAK_DEST}/lxvst
      - ln -s lib/vst3 ${FLATPAK_DEST}/vst3
```

This is all you need to do